### DISCUSSION TITLE
Linkstation 710

### DISCUSSION THREAD

**Discussion by bcoff12** [1000001101000/Debian_on_Buffalo#198] [General]
Has anyone been able to install Debian on Linkstation 710?

I'm looking for a replacement for LS210, which are rock solid reliable but lack performance.  Since I use these as backup targets, performance is secondary to reliability, but it would be nice to shorten the restore window as drives fail in the LS210s.

**Comment by 1000001101000** [comment]
I've got the serial console working on my LS710 and acp_commander etc all work.  I would think you could run Debian 9 or even 10 in a chroot inside the stock firmware without much trouble but I haven't tried that yet. 

The SoC is only partially supported by the mainline linux kernel which complicates things. I haven't managed to boot anything custom on mine yet so am unsure exactly what is needed before I could run a full debian install etc.

My focus lately has been on the TS3010/TS5010/TS3020 devices which I've made some progress on. Once I get something usable for those I'll probably try to port that to the LS500 since they run more or less the same kernel version. After that I might start working on the LS700 though it depends what I learn between now and then.

**Reply by bcoff12** [reply]
Thank you.  The LS700 seems like a solid replacement for my needs - a single drive solution, and on paper it looks far more capable than the LS210s I would be replacing.  I use them to stream single-threaded backups to.  Putting Debian on them gives me a lot of options over the stock firmware, including scripted moves of the backups to a network inaccessible location and (at least on the LS210) NFS support.
